packages/replica-healthcheck/src/index.ts

```diff
@@ -0,0 +1,15 @@
+import * as dotenv from 'dotenv'
+
+import { HealthcheckServer } from './healthcheck-server'
+import { readConfig } from './helpers'
+;(async () => {
+  dotenv.config()
+
+  const healthcheckServer = new HealthcheckServer(readConfig())
+
+  healthcheckServer.init()
+  healthcheckServer.runSyncCheck()
```

Comment:
Does this need an `await`?

Reply:
the thought was that multiple checks could happen synchronously when more is added, does not adding an `await` here make sense in that case?

Reply:
just realized that without the await the app doesn't properly quit on errors (our desired behavior), so added!